controller/search_blackbox_test.go

```diff
@@ -979,7 +979,7 @@ func (s *searchBlackBoxTest) TestUpdateWorkItem() {
 		// given
 		fxt := tf.NewTestFixture(t, s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
 		spaceIDStr := fxt.Spaces[0].ID.String()
-		filter := fmt.Sprintf(`{"label":null}`)
+		filter := fmt.Sprintf(`{"label":{"$EQ":null}}`)
```

Comment:
I have `{"assignee":null}` above. So, I just modified one test with this syntax.